02-microarray/dimension-reduction_microarray_02_umap.Rmd

```diff
@@ -324,17 +329,30 @@ umap_plot
 Our histological subtype groups don't appear to be clustering in a discernible pattern. 
 We could test out other variables as annotation labels to get a further understanding of the cluster behavior of each subgroup.
 
+### Interpretation of UMAP plot and results
```

Comment:
Did you not want to also add this section to the RNA-seq version? (or was it already there and I can't see it)

Reply:
Yup, it is already there: copied it directly!